build_runner/lib/src/entrypoint/options.dart

```diff
@@ -36,6 +37,12 @@ final _defaultWebDirs = const ['web', 'test', 'example', 'benchmark'];
 
 /// Base options that are shared among all commands.
 class SharedOptions {
+  /// A set of explicit filters for files to build.
```

Comment:
Should we make it a set? If we keep it as an Iterable should we change the comment to `/// The explicit filters for files to build.`